jenkins/jobs/teams/jenkins-permissions-job.yaml

```diff
@@ -0,0 +1,202 @@
+  iconClass: icon-jenkins
+  tags: instant-app,jenkins
+  template.openshift.io/long-description: |-
+    The Jenkins Permissions ScheduledJob synchronizes Organization Teams created in GitHub with
+    OpenShift Origin Groups. Special groups in the namespace starting with `jenkins-` will be
+    created to give the members of the GitHub teams team in the local Jenkins instance.
+parameters:
+- name: NAME
+  displayName: Application Name
+  description: The name to use for the objects created by this template
+  value: jenkins-team-sync
+  required: true
+- name: SCHEDULE
+  displayName: Sync Schedule
+  description: The cron tab entry to use for scheduling the Organization Team Sync
+  value: 0 0 * * *
+  required: true
+- name: GROUP_SYNCER_SERVICEACCOUNT
+  displayName: Group Syncer Service Account
+  description: Service account to use as a group syncer
+  value: group-syncer
+  required: true
+- name: SOURCE_REF
+  displayName: Source Ref
+  description: Git reference to use for building the Organization Team Sync image
+  value: master
+  required: false
+- name: SOURCE_URL
+  displayName: Source URL
+  description: Git source URL to use for building the Organization Team Sync image
+  value: https://github.com/openshift/release.git
+  required: true
+- name: SOURCE_CONTEXT
+  displayName: Source Context
+  description: Git source context to use for building the Organization Team Sync image
+  value: jenkins/jobs/teams
+  required: false
+objects:
+- apiVersion: v1
+  kind: ImageStream
+  metadata:
+    labels:
+      app: "${NAME}"
+    name: centos
+  spec:
+    tags:
+    - annotations:
+        openshift.io/imported-from: "centos:7"
+      from:
+        kind: DockerImage
+        name: "centos:7"
+      name: '7'
+- apiVersion: v1
+  kind: ImageStream
+  metadata:
+    labels:
+      app: "${NAME}"
+    name: "${NAME}"
+- apiVersion: v1
+  kind: BuildConfig
+  metadata:
+    labels:
+      app: "${NAME}"
+    name: "${NAME}"
+  spec:
+    output:
+      to:
+        kind: ImageStreamTag
+        name: "${NAME}:latest"
+    source:
+      contextDir: "${SOURCE_CONTEXT}"
+      git:
+        ref: "${SOURCE_REF}"
+        uri: "${SOURCE_URL}"
+      type: Git
+    strategy:
+      dockerStrategy:
+        from:
+          kind: ImageStreamTag
+          name: centos:7
+      type: Docker
+    triggers:
+    - type: ConfigChange
+    - imageChange: {}
+      type: ImageChange
+- apiVersion: v1
+  kind: ServiceAccount
+  metadata:
+    name: "${GROUP_SYNCER_SERVICEACCOUNT}"
+- apiVersion: v1
+  kind: ClusterRole
+  metadata:
+    name: 'group-syncer'
+  rules:
+  - apiGroups:
+    - ""
+    attributeRestrictions: null
+    resources:
+    - groups
+    verbs:
+    - get
+    - update
+- apiVersion: v1
+  kind: ClusterRoleBinding
+  metadata:
+    name: "${GROUP_SYNCER_SERVICEACCOUNT}"
+  roleRef:
+    name: 'group-syncer'
+  subjects:
+  - kind: ServiceAccount
+    name: "${GROUP_SYNCER_SERVICEACCOUNT}"
+    namesapce: ???
```

Comment:
@deads2k @bparees do we have any way to havethis `ClusterRoleBinding` reference the right namespace for this serviceaccount in the template today?